src/lib/learn-client/index.ts

```diff
@@ -37,7 +37,7 @@ export function put(path, token, bodyJson) {
 	})
 }
 
-export function post(path, token, bodyJson) {
+export function post(path, token, bodyJson = {}) {
```

Comment:
Question 🙋: Curious why you added the default for `bodyJson` here?

Reply:
To make it optional 😊